e2e/test/scenarios/custom-column/cc-shortcuts-combine.cy.spec.ts

```diff
@@ -111,26 +111,6 @@ describe("scenarios > question > custom column > expression shortcuts > combine"
       cy.findByLabelText("Separator").should("have.value", "");
     });
   });
-
-  it("should be possible to edit a previous stages' columns when there is an aggregation (metabase#43226)", () => {
```

Comment:
looks to me 1:1 with 

https://github.com/metabase/metabase/blob/master/e2e/test/scenarios/custom-column/cc-shortcuts.cy.spec.ts#L344-L363